.travis.yml

```diff
@@ -51,14 +48,20 @@ matrix:
         packages:
         - gcc-6
         - libssl-dev
-        - libm4ri-dev
     before_install:
       - sh .travis/install-clang-format-linux.sh
   - os: osx
     compiler: clang
-    env: CC_OQS=clang AES_NI=0 USE_OPENSSL=1 ENABLE_CODE_MCBITS=1 ENABLE_NTRU=1 ENABLE_SIDH_IQC_REF=1
+    env: 
+      - CC_OQS=clang 
+      - AES_NI=0 
+      - USE_OPENSSL=1 
+      - OPENSSL_DIR=/usr/local/opt/openssl
+      - ENABLE_CODE_MCBITS=1 
+      - ENABLE_NTRU=1 
+      - ENABLE_SIDH_IQC_REF=1
 before_install:
-      - brew install clang-format openssl libsodium libm4ri-dev
+      - brew install clang-format libsodium
```

Comment:
Shouldn't we keep brew's openssl package since we're referencing it in the OPENSSL_DIR=/usr/local/opt/openssl above?